packages/sit-onyx/src/i18n/locales/en-US.json

```diff
@@ -84,5 +86,21 @@
   "showTooltip": {
     "info": "Show info tooltip",
     "error": "Show error tooltip"
+  },
+  "colorScheme": {
+    "headline": "Change appearance",
```

Comment:
we could add a key just for the word "Appearance" for the user-menu-flyout as well, if that's a common feature in all projects so they don't have to take care of translating it everytime.

Reply:
Currently the translations are only used inside onyx, there is no way for the project to access it.
But I plan to also implement a pre-build user menu option anyways which would inlcude the translation
![image](https://github.com/SchwarzIT/onyx/assets/67898185/ce31a7fb-4dd4-4a8e-a42d-3bf0aa066859)


But we need to wait for #883

Reply:
aah alright, sounds good 😊